### PR TITLE
Rw/#368 dummy ridehail dispath 4ci

### PR DESCRIPTION
**Pull request by carloscaldas-aurea**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/521)

### CODE REVIEW

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
test!

**Comment by carloscaldas** [comment]
Last build: https://beam-ci.tk/job/beam-4ci/1267/

File: src/main/resources/beam-template.conf [hunk 1]
```diff
@@ -6,6 +6,7 @@ beam.agentsim.simulationName = "beamville"
 beam.agentsim.numAgents = 100
 beam.agentsim.thresholdForWalkingInMeters = 100
 beam.agentsim.thresholdForMakingParkingChoiceInMeters = 100
+beam.agentsim.scheduler.maxWindow = "double | 300.0"
```
Comment (wrashid) [inline src/main/resources/beam-template.conf]: remove this and keep schedulerParallelismWindow  instead

File: test/input/beamville/beam.conf [hunk 2]
```diff
@@ -9,6 +9,8 @@ beam.agentsim.simulationName = "beamville"
 beam.agentsim.numAgents = 100
 beam.agentsim.thresholdForWalkingInMeters = 100
 beam.agentsim.thresholdForMakingParkingChoiceInMeters = 100
+beam.agentsim.scheduler.maxWindow = "300.0"
```
Comment (wrashid) [inline test/input/beamville/beam.conf]: remove

**Pull request merged**